src/ethereum/byzantium/vm/instructions/environment.py

```diff
@@ -168,7 +168,7 @@ def calldataload(evm: Evm) -> None:
     # Converting start_index to Uint from U256 as start_index + 32 can
     # overflow U256.
     start_index = Uint(pop(evm.stack))
-    value = evm.message.data[start_index : start_index + 32]
+    value = evm.message.data[start_index: start_index + 32]
```

Comment:
Would you mind running `black` and `isort` as described in [`README.md`](https://github.com/ethereum/execution-specs/blob/master/README.md#development):

```bash
$ pip install -e ".[doc,lint,test]"
$ isort src                         # Organizes imports.
$ black src                         # Formats code.
```